avm/res/net-app/net-app-account/capacity-pool/main.bicep

```
@@ -97,6 +97,8 @@ module capacityPool_volumes 'volume/main.bicep' = [
       subnetResourceId: volume.subnetResourceId
       exportPolicyRules: contains(volume, 'exportPolicyRules') ? volume.exportPolicyRules : []
       roleAssignments: contains(volume, 'roleAssignments') ? volume.roleAssignments : []
+      networkFeatures: contains(volume, 'networkFeatures') ? volume.networkFeatures : 'Standard'
+      zones: contains(volume, 'zones') ? volume.zones : ['1']
```

```suggestion
      networkFeatures: volume.?networkFeatures
      zones: volume.?zones
```

This syntax should actually automatically pull the default from the referenced `volumne` module, effectively saving you one place to maintain :)

---

Agreed, thanks!